### ISSUE TITLE
Making save control label automatically adjust its width when more space is needed due to translation

### ISSUE THREAD

**Issue by Patowhiz** [africanmathsinitiative/R-Instat#6456]
Some languages result into the label of the save control being longer that the text used in English. This forces the design developers to make the width of the save control be longer than needed just incase a translation results in longer label.

@rdstern @lloyddewit I propose to have the save control be dynamic in regards to its width and to detect when the label is not visible enough. This could have been done automatically if the control was a typical VB.NET control but that's not the case.

**Comment by Patowhiz** [comment]
This feature can be done once PR #6434 has been merged.

**Comment by lloyddewit** [comment]
@Patowhiz 

> 
> 
> Some languages result into the label of the save control being longer that the text used in English. This forces the design developers to make the width of the save control be longer than needed just incase a translation results in longer label.
> 
> @rdstern @lloyddewit I propose to have the save control be dynamic in regards to its width and to detect when the label is not visible enough. This could have been done automatically if the control was a typical VB.NET control but that's not the case.

@Patowhiz Thanks for the suggestion, here's my personal view:

I can see the advantages. Possible risks could be:
- if the control expands then it may affect other controls in the dialog
- it may be quite complex to implement and test a solution that copes with all the permutations
- if the user toggles between languages then the dialog controls will shift, this may look strange

I suggest we first try the simpler fixed-width approach and see how it looks. If we decide that the fixed approach looks too ugly when dialogs are displayed in English, then we could consider the variable-width approach above.

**Comment by Patowhiz** [comment]
@lloyddewit yes this can wait. No need to implement this if a simple fixed-width approach works.
